src/plugins/hetero/src/plugin.cpp

```diff
@@ -71,8 +71,15 @@ ov::hetero::Plugin::DeviceProperties ov::hetero::Plugin::get_properties_per_devi
     DeviceProperties device_properties;
     for (const auto& device_name : device_names) {
         auto properties_it = device_properties.find(device_name);
-        if (device_properties.end() == properties_it)
+        if (device_properties.end() == properties_it) {
             device_properties[device_name] = get_core()->get_supported_property(device_name, properties);
+            if (device_name == "CPU") {
+                auto it = device_properties[device_name].find(ov::cache_dir.name());
+                if (it != device_properties[device_name].end()) {
+                    device_properties[device_name].erase(it);
```

Comment:
why do we need to remove it? 
I suppose `ov::Core::compile_model` will filter out `ov::cache_dir` and it will not be passed to CPU finally

Reply:
Filter ov::cache_dir from ov::CoreImpl::query_model to avoid passing it to CPU Plugin

Reply:
I suppose the changes should be made in hetero not core? what's the exact customer case?
if core.query_model("HETERO", ov::cache_dir***) then it's expected to throw exception,
if core.compile_model("HETERO", ov:cache_dir***), then hetero should filter out the cache dir before compile model to target devices.